services/geocoding-v6.js

```diff
@@ -0,0 +1,231 @@
+'use strict';
+
+var xtend = require('xtend');
+var v = require('./service-helpers/validator');
+var pick = require('./service-helpers/pick');
+var stringifyBooleans = require('./service-helpers/stringify-booleans');
+var createServiceFactory = require('./service-helpers/create-service-factory');
+
+/**
+ * Geocoding API service.
+ *
+ * Learn more about this service and its responses in
+ * [the HTTP service documentation](https://docs.mapbox.com/api/search/geocoding-v6/).
+ */
+var GeocodingV6 = {};
+
+var featureTypes = [
```

Comment:
Confirm if we have `street` as a feature type or not

Reply:
@krydima confirmed we have `street` feature type